pkg/apis/batch/validation/validation.go

```diff
@@ -55,19 +54,19 @@ const (
 )
 
 var (
-	supportedPodFailurePolicyActions sets.String = sets.NewString(
-		string(batch.PodFailurePolicyActionCount),
-		string(batch.PodFailurePolicyActionFailJob),
-		string(batch.PodFailurePolicyActionIgnore))
-
-	supportedPodFailurePolicyOnExitCodesOperator sets.String = sets.NewString(
-		string(batch.PodFailurePolicyOnExitCodesOpIn),
-		string(batch.PodFailurePolicyOnExitCodesOpNotIn))
-
-	supportedPodFailurePolicyOnPodConditionsStatus sets.String = sets.NewString(
-		string(v1.ConditionFalse),
-		string(v1.ConditionTrue),
-		string(v1.ConditionUnknown))
+	supportedPodFailurePolicyActions = sets.New(
+		batch.PodFailurePolicyActionCount,
+		batch.PodFailurePolicyActionFailJob,
+		batch.PodFailurePolicyActionIgnore)
+
+	supportedPodFailurePolicyOnExitCodesOperator = sets.New(
+		batch.PodFailurePolicyOnExitCodesOpIn,
+		batch.PodFailurePolicyOnExitCodesOpNotIn)
+
+	supportedPodFailurePolicyOnPodConditionsStatus = sets.New(
+		api.ConditionFalse,
+		api.ConditionTrue,
+		api.ConditionUnknown)
```

Comment:
Pods use `k8s.io/kubernetes/pkg/apis/core.ContainerStatus` not `k8s.io/api/core/v1.ContainerStatus` as containerStatuses.